platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCNotificationIntranetSpaceInvitationTestIt.java

```diff
@@ -0,0 +1,481 @@
+package org.exoplatform.platform.qa.ui.platform.social.functional;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAddManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.*;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceHomePage;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceSettingManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.UserProfilePage;
+import org.exoplatform.platform.qa.ui.social.pageobject.AddUsers;
+import org.exoplatform.platform.qa.ui.social.pageobject.IntranetNotification;
+import org.exoplatform.platform.qa.ui.social.pageobject.MyNotificationsSetting;
+import org.exoplatform.platform.qa.ui.social.pageobject.NotificationActivity;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static com.codeborne.selenide.Condition.not;
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.refresh;
+import static org.exoplatform.platform.qa.ui.core.PLFData.DATA_PASS2;
+import static org.exoplatform.platform.qa.ui.core.PLFData.DATA_USER1;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomString;
+import static org.exoplatform.platform.qa.ui.selenium.locator.ActivityStreamLocator.ELEMENT_SPACE_MENU_ACTIVITY_PORTLET;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+
+@Tag("functional")
+@Tag("social")
+public class SOCNotificationIntranetSpaceInvitationTestIt extends Base {
```

Comment:
class name must be "SOCNotificationIntranetSpaceInvitationTestIT" not "SOCNotificationIntranetSpaceInvitationTestIt"

Reply:
Corrected